api-scanner/src/main/java/net/corda/plugins/ScanApi.java

```diff
@@ -434,6 +450,23 @@ private static boolean isValid(int modifiers, int mask) {
         return (modifiers & mask) == modifiers;
     }
 
+    private boolean isExcluded(ApiPrintWriter writer, MethodInfo methodWithoutAnnotations) {
+        final StringWriter stringWriter = new StringWriter();
```

Comment:
You have formatted the method signature nicely for use in `build.gradle`, but would this be more stable if we used the method's descriptor?
```java
String className = methodWithoutAnnotations.getClassName();
String methodDescriptor = methodWithoutAnnotations.getTypeDescriptorStr();
return this.excludeMethods.containsKey(className) &&
                this.excludeMethods.get(className).contains(methodDescriptor);
```
The method descriptor is the stable part of the ABI that is written into the bytecode, whereas our formatting of it may change.

Reply:
For reference,
```java
<init>(int, String, int, kotlin.jvm.internal.DefaultConstructorMarker)
```
has a descriptor of
```java
(ILjava/lang/String;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
```